app/policies/cinema_hall_policy.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class CinemaHallPolicy < ApplicationPolicy
+  attr_reader :user, :cinema_hall
+
+  def initialize(user, cinema_hall)
+    super(user)
+    @cinema_hall = cinema_hall
+  end
```

Comment:
Consider [this](https://github.com/atytko/monte-cinema/pull/29#discussion_r699987474) comment